servicecontrol/installation.md

```diff
@@ -18,6 +18,8 @@ The ServiceControl Installation has the following prerequisites:
 
 If ServiceControl is installed via the Platform Installer then the installation and configuration of these prerequisites are managed by the installer.
 
+NOTE: ServiceControl should be installed on a separate (virtual) machine in a production environment to isolate the audit and error queue message ingestion. These operations have a big impact on other processes. ServiceControl must be given sufficient CPU, RAM (6GB minimum) and storage suitable for low latency write operations, similar to how a database server would be provisioned. See [ServiceControl Capacity Planning](capacity-and-planning.md) for more guidance.
```

Comment:
I don't like the "(virtual)" as it reads like VMs are a recommendation. 

I would use the terms "dedicated machine" and "dedicated storage" in here to emphasize.

Maybe tie these two points together by suggesting the machine can be virtual but the storage should be dedicated to ServiceControl?